utils/getInvoices/tests/getInvoices.placing.test.ts

```diff
@@ -5,139 +5,892 @@ import { expect } from '@jest/globals'
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should NOT load when service = { rentPrice: 10 }, company = null', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should NOT load when service = null, company = { totalArea: 10, inflicion: true }', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = {
+        totalArea: 10,
+        inflicion: true,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 10, inflicion: false }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 10,
+        inflicion: true,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        price: 100,
+        sum: 100,
+      })
+    })
+
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 10, inflicion: true }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 15,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 12,
+        inflicion: true,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        price: 180,
+        sum: 180,
+      })
     })
   })
 
-  it('should load Placing price from payment as sum', () => {
-    const company: Partial<IRealestate> = null
-    const service: Partial<IService> = null
-    const payment: Partial<IPayment> = {
-      invoice: [{ type: ServiceType.Placing, price: 0, sum: 100 }],
-    }
-    const prevPayment: Partial<IPayment> = null
+  describe('props: { service, company, prevPayment }', () => {
+    it('should NOT load when service = null, company = null, prevPayment = null', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const prevPayment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should NOT load when service = null, company = null, prevPayment = { invoice: [Inflicion] }', () => {
```

Comment:
rename to `should NOT load when service = null, company = null, prevPayment = { invoice: [Placing] }`